.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2
+jobs:
+  build_test_push:
+    docker:
+      - image: docker:stable
+    environment:
+      BUILD_IMAGE: antenna:build
+      TAGGED_IMAGE: mozilla/antenna
```

Comment:
Socorro uses this image name, but we can switch it. Is there anything else using this image name?

Reply:
I doubt it, I imagine we're the only consumers of this image.